README.md

```diff
@@ -1,7 +1,34 @@
 # 2048
+
 A HashiCorp reskin of [Gabriele Cirulli's version of 2048](https://github.com/gabrielecirulli/2048).
 
 Playable [here](https://tunzor.ca/hashi-2048)
 
-### Screenshot
+## Screenshot
 ![2048 screenshot](images/screen-cap.jpg)
+
+## Docker Image
+
+A Docker Image of 2048 can be built with the following `Dockerfil`:
+
+```dockerfile
+FROM alpine:latest
+
+MAINTAINER Anthony Russo <github.com/tunzor>
+
+RUN apk --update add nginx
+
+COPY ./2048 /usr/share/nginx/html
+
+EXPOSE 80
+
+CMD ["nginx", "-g", "daemon off;"]  
```

Comment:
this dockerfile works

```suggestion
FROM nginx:latest

COPY . /usr/share/nginx/html

EXPOSE 80

CMD ["nginx", "-g", "daemon off;"]
```

Reply:
This works and is cleaner! I was in a time crunch, so I based it off of the [upstream Dockerfile](https://github.com/alexwhen/docker-2048/blob/master/Dockerfile)